javascript/utils/RepeatAttackCore.js

```diff
@@ -112,11 +117,10 @@ function findWeapons(
   return mapped.map((p) => p.planet).slice(0, numOfPlanets);
 }
 function planetIsRevealed(planetId) {
-  return !!df.getLocationOfPlanet(planetId);
+  return !!contractsAPI.getLocationOfPlanet(planetId);
```

Comment:
Wait, do you have direct access to `contractsAPI` or does that need to be `df.contractsAPI`?

Reply:
Good catch, you're right